### DISCUSSION TITLE
About the publii homepage theme

### DISCUSSION THREAD

**Discussion by GianlucaLocri** [GetPublii/Publii#1557] [Help]
Hi, new publii user here!
I just noticed that (obviously) the publii website is made with publii.
I just wonder what theme did they used and if such theme (or similar) is available for download.
I think it can be an useful resource, also because all the available themes are for editorial or portfolio purposes... If such a theme would be available, It would be very useful to present a product, or to create a small company website (like the one I'm trying to do)
Thanks for any advice!

**Comment by 3muchfun** [comment]
Hi, @GianlucaLocri , you may find all available themes here: [(https://marketplace.getpublii.com/themes/)](https://marketplace.getpublii.com/themes/) . In case of none of them fulfill your needs, I am afraid you have to build your own theme, following the [Developer Docs](https://getpublii.com/dev/)

**Comment by candidexmedia** [comment]
This question has been raised before (might've been in the old forum?), so you're not alone in your feelings.

The thing is, unless you have some intermediate HTML + CSS skills, recreating a home page like that one would be tricky, even with a theme.

Building complex, custom page layouts inside of posts isn't Publii's strong suit. See, for instance, this old forum post [asking how to add a simple three column layout](https://forum.getpublii.com/topic/front-page-index-how-to-make-a-section-with-3-responsive-column/#post-11731).

Imagine now for layouts like these ones...

![image](https://github.com/GetPublii/Publii/assets/95003834/ac462100-e56f-49eb-a8f5-568e550901ce)

![image](https://github.com/GetPublii/Publii/assets/95003834/f0b3523a-2566-49ee-8eb9-c338c0b3720b)


Creating one-off layouts [like the solution I shared earlier](https://forum.getpublii.com/topic/front-page-index-how-to-make-a-section-with-3-responsive-column/#post-11731) are one thing, but to maintain consistent styling, you need reusable components.

It would require either hard-coding certain sections, which wouldn't give users a lot of flexibility in terms of which layout elements go where, OR would require some customizations of the WYSIWYG Editor (with the [template plugin](https://gist.github.com/candidexmedia/367711a80274b4c005fd5983df97d20c#file-2-adding-a-template-plugin-for-reusable-chunks-md)) and/or handlebars theme helpers (see [docs](https://getpublii.com/dev/how-to-create-custom-handlebars-helpers/#how-to-create-custom-handlebars-helpers) and [old forum post](https://forum.getpublii.com/topic/global-string-variable/)).

That said, if you enjoy tinkering like I do, theme development can be rewarding. I've learned so much from customizing my Publii themes. Let me know if you'd like more tips on this.
